### ISSUE TITLE
Metagenomics Project: "Find and download the data" SCP code 

### ISSUE THREAD

**Issue by nkavaler1** [dib-lab/dib_rotation#55]
doing it the way it was said resulted in error:
```
nataliekavaler@Natalies-MacBook-Air-18 ~ % scp -P 2022  -i /.ssh/id_rsa nkavaler@farm.cse.ucdavis.edu:2020_rotation_project/raw_data/*.html . 
zsh: no matches found: nkavaler@farm.cse.ucdavis.edu:2020_rotation_project/raw_data/*.html

```
fixed version
```
nataliekavaler@Natalies-MacBook-Air-18 ~ % scp  -i /.ssh/id_rsa nkavaler@farm.cse.ucdavis.edu:2020_rotation_project/raw_data/\*.html . 
Warning: Identity file /.ssh/id_rsa not accessible: No such file or directory.
SRR1976948_2_fastqc.html                          100%  591KB   4.3MB/s   00:00 
```

**Comment by mr-eyes** [comment]
You can also do
```
scp  -i /.ssh/id_rsa "nkavaler@farm.cse.ucdavis.edu:2020_rotation_project/raw_data/*.html" .
```

**Comment by mr-eyes** [comment]
Note the `~` before `/.ssh` and the quotes between the server path of wildcards.

**Comment by ctb** [comment]
fixed by #57 I believe

**Comment by mr-eyes** [comment]
> fixed by #57 I believe

I don't think the wildcard issue has been resolved.
https://github.com/dib-lab/dib_rotation/pull/57/files#diff-54ddbdd9add2bd1c3b5c985ebd8fb673eda4ea4ed4bc4913eb3ec468fab2b801R201
